apollo-federation/src/sources/connect/json_selection/README.md

```diff
@@ -89,10 +89,11 @@ NamedPathSelection   ::= Alias PathSelection
 NamedFieldSelection  ::= Alias? Key SubSelection?
 NamedGroupSelection  ::= Alias SubSelection
 Alias                ::= Key ":"
-PathSelection        ::= (VarPath | KeyPath | AtPath) SubSelection?
+PathSelection        ::= (VarPath | KeyPath | AtPath | ExprPath) SubSelection?
 VarPath              ::= "$" (NO_SPACE Identifier)? PathStep*
 KeyPath              ::= Key PathStep+
 AtPath               ::= "@" PathStep*
+ExprPath             ::= "(" LitExpr ")" PathStep*
```

Comment:
oh good, parsing will enforce a single expression here. that's actually a nice improvement over echo()!

Reply:
indeed!